runtime/JavaScript/src/antlr4/dfa/DFA.js

```diff
@@ -104,7 +104,7 @@ DFA.prototype.setPrecedenceStartState = function(precedence, startState) {
 
 DFA.prototype.setPrecedenceDfa = function(precedenceDfa) {
 	if (this.precedenceDfa!==precedenceDfa) {
-		this._states = new DFAStatesSet();
+		this._states = new Set();
```

Comment:
suggest to drop setPrecedenceDfa altogether, method is unused, and now deprecated/throws in reference Java runtime

Reply:
Done.